docs/build/_toc.json

```diff
@@ -8,6 +8,10 @@
     {
       "title": "Build circuits with Qiskit",
       "children": [
+        {
+          "title": "Circuit library",
+          "url": "/build/circuit-library"
+        },
```

Comment:
FYI @frankharkins -- I moved the Circuit library first in its section since that's where @javabster has it listed in the [Build sidebar issue](https://github.com/Qiskit/documentation/issues/100), but feel free to change it back if it was meant to be second in the list!